packages/product-json-to-csv/src/main.js

```diff
@@ -197,7 +197,7 @@ export default class ProductJsonToCsv {
     )
   }
 
-  async _resolveVariantReferences(variant: Variant) {
+  async _resolveVariantReferences(variant: Variant): Promise<Variant> {
```

Comment:
Flow requires exported functions/interfaces to have an explicit return type declaration

```
Cannot build a typed interface for this module. You should annotate the exports of this module with types. Missing type annotation at function return:Flow(signature-verification-failure)
```

Reply:
Makes somehow sense to be explicit about the return type.